_episodes/14-runtime.md

```diff
@@ -22,6 +26,12 @@ rather than the command line parameters.  To do this, use
 ~~~
 {: .source}
 
+Any [expressions](../13-expressions) like `$(inputs.message)` are expanded by the CWL engine before creating the file; here inserting the value at the input `message`. 
```

Comment:
How do I link to the previous step?

Reply:
Full URL seems to be what we have done so far...

Reply:
I see `index.html` uses `./14-runtime/index.html` -- as much as it pains me to see `index.html` on the web I guess style `../13-expressions/index.html` here then.